getting-started/index.html

```diff
@@ -63,6 +63,8 @@ <h4>Steam</h4>
                         <div class="terminal">/Applications/Steam.app/Contents/MacOS/steam_osx -console</div>
                         <p>You should now see additional tab in the top menu called CONSOLE. Open it, and enter the following command.
                         <div class="terminal">download_depot 285330 285331</div>
+                        <p>If you also own RCT1 Deluxe on steam and want those assets as well, you could enter this command. You don't have to wait for the first download to finish before entering the second one.
+                        <div class="terminal">download_depot 285310 285312</div>
```

Comment:
On the wiki, I used `download_depot 285310 285312`. I don't know what the difference is between these two depots, do you happen to know?

Reply:
Oops, I'm pretty sure I checked yesterday but picked the wrong one still.

https://steamdb.info/app/285310/depots/

The *11 one is the correct one, the *12 one appearantly has ~200MB of extra stuff in German.